orc8r/gateway/python/magma/common/service_registry.py

```diff
@@ -85,6 +86,31 @@ def reset():
         """
         ServiceRegistry.get_registry()["services"] = {}
 
+    @staticmethod
+    def get_bare_bootstrap_rpc_channel(domain_name, port, rootca_path=None):
+        """
+        Return an RPC channel to bootstrap service in CLOUD, without referencing a
+        control proxy config.
+
+        Returns:
+            grpc channel
+        """
+        rootca_path = ROOTCA_PATH if not rootca_path else rootca_path
```

Comment:
So if no rootca path is passed in, we want to use the default system credentials. I *think* that means you don't have to pass any ssl creds, but not sure how it works. But if you want, we can leave it like this for now, and I'll test it when I get around to making the Orc8r<>Let'sEncrypt work e2e